nssp_demo/batch/setup_job.py

```diff
@@ -63,14 +72,27 @@ def main(job_id, pool_id, container_image) -> None:
         .to_list()
     )
 
-    for disease in ["COVID-19", "Influenza"]:
-        for state in all_states:
-            task = get_task_config(
-                f"{job_id}-{state}-{disease}",
-                base_call=base_call.format(state=state, disease=disease),
-                container_settings=container_settings,
-            )
-            client.task.add(job_id, task)
+    report_dates = [
+        datetime.date(2023, 10, 11) + datetime.timedelta(weeks=x)
+        for x in range(30)
+    ]
```

Comment:
I would prefer a polars date_range so we can more easily discern/configure the end date.

Reply:
In this case I think a number of weeks is fairly useful to reason about, but we can change going forward.